app/controllers/observations_controller/destroy.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# see observations_controller.rb
+module ObservationsController::Destroy
+  # Callback to destroy an observation (and associated namings, votes, etc.)
+  # Linked from: observations/show (note that links require jquery_ujs)
+  # Inputs: params[:id] (observation)
+  # Redirects to next obs in query or :index.
+  def destroy # rubocop:disable Metrics/AbcSize
```

Comment:
Suggestion: Leave a short explanation whenever disabling a cop so we'll know why in the future.
Here It could simply be a reference to a comment in this PR like:
```suggestion
  # cop disabled per https://github.com/MushroomObserver/mushroom-observer/pull/1060#issuecomment-1179410808
  def destroy # rubocop:disable Metrics/AbcSize
```
(Note that rubocop won't complain about url's extending the line more than 80 characters.)

Reply:
I'm not actually sure we should be disabling ABC unless we mean for it to be permanently disabled, no?  As I understand it, the complaint is legitimate, but the cure it outside the scope of the PR.  I'd rather the cop still report the error in such cases.